nmdc_runtime/api/endpoints/util.py

```diff
@@ -442,6 +442,7 @@ def persist_content_and_get_drs_object(
                     ),
                     "access_methods": [{"access_id": drs_id}],
                 },
+                timestamp= datetime.now(tz=ZoneInfo('America/Los_Angeles')).isoformat(timespec='minutes')
```

Comment:
This change makes sense to me. 

A `string` is being generated that indicates the current time (in Los Angeles, CA) down to the minute.

[Docs](https://docs.python.org/3/library/datetime.html#datetime.datetime.isoformat) for `datetime.isoformat()` say it returns a `string` representation of the time in ISO 8601 format.

![image](https://github.com/microbiomedata/nmdc-runtime/assets/134325062/5a8662ba-8958-458f-b410-648351cdf92b)

Reply:
I recommend removing the space after the equals sign, since this is "argument setting" as opposed to "variable assignment" (I think that's a [Python convention](https://peps.python.org/pep-0008/#other-recommendations)). 

```diff
- timestamp= datetime.now(...
+ timestamp=datetime.now(...
```